transport/http/pom.xml

```diff
@@ -102,6 +102,12 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.kaazing</groupId>
+            <artifactId>gateway.service.http.directory</artifactId>
```

Comment:
Hi Alex,

This change here is wrong. By introducing it you created a cyclic dependency. The artifact gateway.service.http.directory depends on gateway.transport.http as well. So maven will not know which to build first.

Best regards,
Doru